.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Pull Request Agreement
+
+By opening this pull request, you agree to the following:
+
+- [ ] I have read the [contributing](http://github.com/infinitered/ignite-cookbook/blob/main/CONTRIBUTING.md) guidelines.
+- [ ] I have adhered to the coding standards and best practices.
```

Comment:
Not a blocker:

I was confused by this at first, I didn't think we had any existing contributing guidelines, and then I realized we had that doc that was written for IR contributions. I was also expecting this to be primarily about recipes and what a quality recipe looks like, so I was surprised to see line items about "coding standards" and "code changes".

I think we should clearly separate documentation for contributing recipes from contributing changes to the website. This can be separate headers in the contributing doc, it should just be easy to find which parts each use case would care about, and they should have different line items in the checklist, e.g.,:

```
If you are contributing a recipe:
...checklist...

If you are contributing changes to the cookbook website:
...checklist...
```

What do you think @cdanwards ?

Reply:
In your opinion, does this extend to the `Contributing` document? That's focused on recipe contributions at the moment.

Reply:
Yeah, I think so. I think they can be separate documents or one doc split in two sections.

Reply:
Okay, great. I'll update these when I have some time tomorrow.

Reply:
So the more I'm looking at this, I think the PR template checklist might be overkill. Likely we just want to put in the Template that recipe contributions are under CC BY 4.0 license.